src/utils/types.ts

```diff
@@ -53,3 +53,7 @@ export interface ChatRoom {
     subscribeMessages: (callback: (message: ChatMessage) => void) => void
     subscribeMetadata: (callback: (metadata: any) => void) => void
 }
+
+export enum StorageKey {
+    EncryptedSessionKey = 'streamr-chat-encrypted-session-key',
```

Comment:
@bearni95 what's your opinion on using PascalCase for enums? It's more typescriptish. 😉  Would you mind if we sticked to it?

Reply:
Sounds good to me, I'll stick to those!